examples/with-linaria/README.md

```diff
@@ -1,6 +1,6 @@
-# Example app with [linaria](https://linaria.now.sh/)
+# Example app with [linaria](https://linaria.dev/)
 
-This example features how to use [linaria](https://linaria.now.sh/) as the styling solution instead of [styled-jsx](https://github.com/zeit/styled-jsx).
+This example features how to use [linaria](https://linaria.dev/) as the styling solution instead of [styled-jsx](https://github.com/zeit/styled-jsx).
```

Comment:
```suggestion
This example features how to use [linaria](https://linaria.dev/) as the styling solution instead of [styled-jsx](https://github.com/vercel/styled-jsx).
```